portal-web/test/functional/com/liferay/portalweb/macros/PermissionsInline.macro

```diff
@@ -67,6 +67,29 @@ definition {
 			value1 = "Your request completed successfully.");
 	}
 
+	macro addOrRemoveSubscribePermissionFromAllRoles {
```

Comment:
I don't really like the name of this macro but since other ones follow this format and refactor macro names isn't a priority this year we can work on this in the future